test/e2e/features/global_rule.go

```diff
@@ -56,6 +59,30 @@ spec:
 		// Wait until the ApisixClusterConfig create event was delivered.
 		time.Sleep(3 * time.Second)
 
+		ar := fmt.Sprintf(`
```

Comment:
I think it's worthy to write some documents about "How to access Apache APISIX Prometheus Metrics on Kubernetes".

Reply:
There seems to be a similar article on APISIX's WeChat account

Reply:
Could we also file them?

Reply:
OK, let's do it in another PR.

Reply:
See https://github.com/apache/apisix-ingress-controller/issues/953.